lsmb-request.pl

```diff
@@ -19,7 +19,7 @@ =head1 COPYRIGHT
 =cut
 
 package LedgerSMB::Handler;
-
+use lib '.';
```

Comment:
After some further testing, it looks like FindBin actually works but lib->import() doesn't which is why we saw the breakage.  I think for 1.4.32 we should replace the above with:

```
use FindBin;
use lib $FindBin::Bin;
```